### ISSUE TITLE
Add Branch Info to Activities

### ISSUE THREAD

**Issue by gmedici** [scm-manager/scm-manager#1726]
#### Problem to be solved

For a GIT repository in the Activities view there is no indication on the branch upoun witch the commit was pushed. 
Moreover this information is not present nor in the "Details" of a commit nor in the "Sources".

#### Proposed solution

Add the name of the branch as a link in all those views allowing to navigate to the repository code sources specific branch.

As a sidenote, I would suggest to consider ordering the branches in the select box of the repository code view based on the time of the last commit on each branch, putting the last actives on top. 
It could be usefull to quickly discover the most active branches.

**Comment by pfeuffer** [comment]
Thanks for the suggestions. Showing branches for commits is a good point and we've put it into our backlog.

Regarding the branch dropdown: Do you find the grouping in the branch overview helpful? Would these two groups meet your suggestion for the dropdown?

**Comment by gmedici** [comment]
Yes after I've written the suggestion I discover that in brach overview there is already a sorting of the branches based on activity.
I think that's very good. 
And probably more than enought, instead of sorting the dropdown.
